src/components/login.tsx

```diff
@@ -0,0 +1,35 @@
+import { FormEvent, useState } from "react";
+
+function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (e: FormEvent) => {
+    e.preventDefault();
+    // Add login logic here
+  };
+
+  return (
```

Comment:
**suggestion**: Add type annotations for the event parameter to enhance type safety and readability.